### PR TITLE
Direct integration into git

### PR DESCRIPTION
**Pull request by ASPePeX**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,7 +100,8 @@ _to be written_
 (support for different workflows)
 
 # Integration into existing services
-
+## Integration: git
+proposing _git-issue(s)_: in the way github pages uses a specific branch or [git-flow](https://danielkummer.github.io/git-flow-cheatsheet/) abstracts the handling of branches, _git issue(s)_ uses a specified orphaned branch and is the users interface to managing issues.
```
Comment (matthiasbeyer) [inline README.md]: Just to be on the same page here: You're not talking about [softwarescales/git-issues](https://github.com/softwarescales/git-issues) here, do you?

Comment (ASPePeX) [inline README.md]: I'm not, but I didn't know about this project even tho I was looking for something like that. Anyways, I propose a different approach, not using github or bitbuckets issue system but building one that can be used within the repository itself, so everything is in one spot.

Comment (matthiasbeyer) [inline README.md]: > not using github or bitbuckets issue system but building one that can be used within the repository itself, so everything is in one spot.

That's the general idea, yes.

From the top of the document (on master):

```
This repository contains [...] how one could implement distributed issue tracking with git.
```

Comment (matthiasbeyer) [inline README.md]: Short followup comment: We have a pretty complex idea in mind (me and @TheNeikos), but it is not written down yet.

We will start a PR to discuss and so on, soonish (as soon as @TheNeikos gets his stuff together to write the ideas down).

Comment (ASPePeX) [inline README.md]: With my addition I'm proposing a possibility how it could be handled, branches and a CLI interface that handles all the funky stuff.

**Pull request closed without merging**